src/core/tests/type_prop/multinomial.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/multinomial.hpp"
+
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/op/constant.hpp"
+
+TEST(type_prop, multinomial_i64_i32) {
```

Comment:
Can be converted to `TEST_F`, but requrieds to used Test fixture e.g
`class TypePropMultinominalV13Test : public TypePropOpTest<ov::op::v13::Multinominal>{};`
The helper class TypePropOpTest form common_test_utils/type_prop.hpp have helper function to create tested operator `make_op` which can simplyfy making test op and is easy to change it version if required.

Reply:
Changed to use TEST_F!